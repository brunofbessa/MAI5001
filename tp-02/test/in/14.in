2
5
-5,3,-1,0,9,12,15,-3,X
2,-4,6,-9,10,7,X
1,2,3,4,5,6,7,8,9,10,X
-1,-2,-3,-4,-5,-6,-7,-8,-9,-10,X
-50,50,-25,25,-15,15,X