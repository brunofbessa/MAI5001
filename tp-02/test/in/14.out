-50,-25,-15,-10,-9,-9,-8,-7,-6,-5,-5,-4,-4,-3,-3,-2,-1,-1,0,1,2,2,3,3,4,5,6,6,7,7,8,9,9,10,10,12,15,15,25,50